census/src/test/java/io/grpc/census/CensusModulesTest.java

```diff
@@ -716,13 +717,14 @@ private void assertRealTimeMetric(
     assertEquals(expectedValue, record.getMetricAsLongOrFail(measure));
   }
 
-  private void assertZeroRetryRecorded() {
+  private void assertPerCallMetrics(double expectedLatencyValue) {
```

Comment:
I see this is used to test the new metric but I don't see any of the retry tests (e.g. `recordRetryStats` on line 515) testing the new metric (total call latency which should be the sum of all retries latencies). May be you can update the `recordRetryStats` test?

Reply:
Updated